libse/SubtitleFormats/SubtitleFormat.cs

```diff
@@ -363,9 +363,9 @@ public static int MillisecondsToFramesMaxFrameRate(double milliseconds)
             return frames;
         }
 
-        public static int FramesToMilliseconds(double frames)
+        public static double FramesToMilliseconds(double frames)
         {
-            return (int)Math.Round(frames * (TimeCode.BaseUnit / Configuration.Settings.General.CurrentFrameRate));
+            return Math.Round(frames * (TimeCode.BaseUnit / Configuration.Settings.General.CurrentFrameRate));

```

Comment:
Return to double and let any subtitle format calling this method to deal with "cast" individually.